### ISSUE TITLE
POST indicator spins forever

### ISSUE THREAD

**Issue by Jarry1250** [hatnote/montage#7]
Every 3-4 'votes', the loading indicator will just spin continuously without resolution. This is despite the POST appearing to complete successfully (according to Firebug).

All Firebug reports is
```
TypeError: dbg is undefined
chrome://firebug/content/debugger/debuggerLib.js
Line 556
```
...which doesn't seem very helpful.

I'm using an update to date version of Firefox (beta in fact). Mobile Firefox was working for me earlier and I'll try in Chrome now.

**Comment by Jarry1250** [comment]
Ah, Chrome developer tools gives something more useful:

```
angular.js:13920 TypeError: Cannot read property 'entry' of undefined
    at controller.getImageName (round.js:57)
    at fn (eval at compile (angular.js:14817), <anonymous>:4:398)
    at angular.js:15946
    at Scope.$digest (angular.js:17515)
    at Scope.$apply (angular.js:17790)
    at done (angular.js:11831)
    at completeRequest (angular.js:12033)
    at XMLHttpRequest.xhr.onload (angular.js:11966)
```

I'm sure I wasn't getting this problem earlier though on Mobile Firefox

**Comment by yarl** [comment]
This one should be fixed by bd9c3491ad3c837f4e040307845527c8515b985f. Please reopen if persist.
